### PR TITLE
Play button on the video add (edit_memory) (fix #54)

### LINKED ISSUE #54: [BUG] Video's are not indicated in the EditMemoryActivity
**Describe the bug**
In `EditMemoryActivity` you don't see what media is an image and what is a video. This should be indicated by a play icon on the video's.

### PR DESCRIPTION
**Pull request by Hicham010**
I added a play button to the video in the edit_memory_activity

Fix #54

### CODE REVIEW

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/adapter/ViewPagerAdapter.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void onBindViewHolder(@NonNull final PagerVH holder, int position) {
                 public void onClick(View v) {
                     Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(((Video) m).getVideoPath()));
                     intent.setDataAndType(Uri.parse(((Video) m).getVideoPath()), "video/mp4");
-                    holder.pageImage.getContext().startActivity(intent);
+                    holder.pageImage.getContext().startActivity(intent) ;
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/adapter/ViewPagerAdapter.java]: This space is not necessary. Remove it.

**Review by MrTheGood — CHANGES_REQUESTED** [review]
Seems to work. Nice.

File: app/src/main/res/layout/item_memory_video_edit.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_margin="6dp"
+        android:background="#88000000"
+        android:scaleType="centerInside"
+        android:src="@drawable/ic_play_video"
+        tools:layout_editor_absoluteX="5dp"
+        tools:layout_editor_absoluteY="5dp"
+        />
+
```
Comment (Almightywesley) [inline app/src/main/res/layout/item_memory_video_edit.xml:30]: Space is nod needed

**Review by Almightywesley — APPROVED** [review]
i see that you have implemented the feedback i gave you nice job

File: app/src/main/res/layout/item_memory_video_edit.xml (hunk 2 above)
Comment (MrTheGood) [reply app/src/main/res/layout/item_memory_video_edit.xml:30]: Yes it is.

**Review by MrTheGood — APPROVED** [review]

**Pull request merged**